### ISSUE TITLE
[Feature Request]: Add Image save option for easier file management

### ISSUE THREAD

**Issue by LiJT** [diStyApps/ComfyUI-disty-Flow#35]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### Summary

Please add this feature on Flow UI, if that isnt too hard. my ComfyUI output folder is packed with newly generated image, if I generated a ton of image it is very hard to manage all of the files

### Description

![PixPin_2024-12-13_19-53-08](https://github.com/user-attachments/assets/1cefc6d5-1aec-4fbc-acde-b39e8dcf3f3c)
Is it possible we can have sort of menu in the UI, so user can toggle on or off if they wish to save the image or not? And User also can input their image filename prefix to save the output image in a folder structure if they like?

**Comment by diStyApps** [comment]
Yes, but this is bigger part planed for flow The galleries, the model and image galleries, so they will work together it is an entire system. but it is planed. there a lot features to add.

i really like your prototyping, thank you for your contributions.

**Comment by LiJT** [comment]
> Yes, but this is bigger part planed for flow The galleries, the model and image galleries, so they will work together it is an entire system. but it is planed. there a lot features to add.
> 
> i really like your prototyping, thank you for your contributions.

Wow awesome! cannt wait for the next major release!
